src/language/sv.json

```
@@ -1,8 +1,10 @@
 {
+    "name": "Svenska",
```

```suggestion
    "languageName": "Svenska",
```
to specify a bit more. Name for me is a persons name. Or maybe just use "language".